package/scie-pants.lift.json

```diff
@@ -114,14 +114,14 @@
               "--wait-for-client",
               "{scie.bindings.install:VIRTUAL_ENV}/bin/pants",
               "--pants-bin-name={scie.env.PANTS_BIN_NAME}",
-              "--pants-version={scie.env.PANTS_VERSION={scie.bindings.install:PANTS_VERSION}}",
               "{scie.bindings.install:PANTS_SHA_FIND_LINKS}"
             ]
           },
           "bootstrap-tools": {
             "description": "Introspection tools for the Pants bootstrap process.",
             "env": {
-              "=PEX_ROOT": "{scie.bindings}/pex_root"
+              "=PEX_ROOT": "{scie.bindings}/pex_root",
+              "=PEX_PYTHON_PATH": "{scie.files.{scie.env.PYTHON}-{scie.platform}}/python/bin/{scie.env.PYTHON}"
             },
             "exe": "{scie.files.{scie.env.PYTHON}-{scie.platform}}/python/bin/{scie.env.PYTHON}",
             "args": [
```

Comment:
Just one small question about `bootstrap_tools.py`, looking for confirmation: on my read of that file in isolation, it _seemed_ to make `bootstrap-cache-key` much harder to use: specifying the `--python-distribution-hash` and `--pants-version` arguments externally seems hard/impossible. However, my read of this section is that users won't have to set them directly because invocations are expected to go via this entry point, and thus have them set automatically.

That is, `PANTS_BOOTSTRAP_TOOLS=1 scie-pants bootstrap-cache-key` (or similar) will be invoking the actual bootstrap tools `main` with the appropriate args, transparent to users?

Reply:
Where the PANTS_VERSION env var is set by this code in the main scie-pants launcher binary just before it re-executes itself to satisfy the request:
https://github.com/pantsbuild/scie-pants/blob/dfb3961cf7122e1ae296e98b80da24164903313c/src/main.rs#L112-L118
https://github.com/pantsbuild/scie-pants/blob/dfb3961cf7122e1ae296e98b80da24164903313c/src/main.rs#L168-L176
https://github.com/pantsbuild/scie-pants/blob/dfb3961cf7122e1ae296e98b80da24164903313c/src/main.rs#L143-L152

Reply:
Very cool. Thanks for confirming!